src/components/product/ProductGrid/ProductGrid.tsx

```diff
@@ -15,7 +15,13 @@ function ProductGrid({ products, page, pageSize }: Props) {
     <ul className="temp-grid-row">
       {products.map((product, idx) => (
         <li key={`${product.id}`}>
-          <ProductCard product={product} index={pageSize * page + idx + 1} />
+          <ProductCard
+            product={product}
+            index={pageSize * page + idx + 1}
+            showActions={false}
+            bordered
+            outOfStock={product.offers.offerCount < 1}
```

Comment:
Interesting! In @israelswf's #221 he used `offers.availability` to check for this. Do `offers.offerCount` and `offers.availability` have the same meaning in the end? If they're the same, I wonder if we can pick just one of them.

Reply:
Since we alreaty have [this mapping](https://github.com/vtex/faststore/blob/653c9d8e54558c19b9ee7aa8c3387d56ed2d16bb/packages/api/src/platforms/vtex/resolvers/offer.ts#L14-L17) in the `faststore`, I think it is a good strategy to check the `availability` instead of the `offerCount`.